apps/server/src/schemas/recipient.schema.ts

```diff
@@ -0,0 +1,12 @@
+import { Prop, Schema, SchemaFactory } from '@nestjs/mongoose';
+import { HydratedDocument } from 'mongoose';
+
+export type RecipientDocument = HydratedDocument<Recipient>;
+
+@Schema()
+export class Recipient {
```

Comment:
actually this is the recipient model according the API definition, could you update the schema?

`  {
      "id": "string",
      "email": "string",
      "first_name": "string",
      "last_name": "string",
      "created_at": "2024-06-06T14:53:55.703Z",
      "unsubscribed": true
    }`